core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
@prasanna08 You think we can pass a standard message dict defined in fecund for this purpose?